lab_4/main.py

```diff
@@ -4,16 +4,35 @@
 """
 
 from typing import Tuple
-from lab_4.storage import Storage
 from lab_4.language_profile import LanguageProfile
+from lab_4.storage import Storage
 
 
 # 4
 def tokenize_by_letters(text: str) -> Tuple or int:
     """
     Tokenizes given sequence by letters
     """
-    pass
+    if not isinstance(text, str):
+        return -1
+    words = []
+    invaluable_trash = ['`', '~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '-', '+',
```

Comment:
If you are not going to change the array, it is better to define it as a `tuple`.